pkg/api-server/resource_endpoints_test.go

```diff
@@ -1037,4 +1038,8 @@ var _ = Describe("Resource Endpoints", func() {
 		Expect(test_metrics.FindMetric(metrics, "api_server_http_requests_inflight")).ToNot(BeNil())
 		Expect(test_metrics.FindMetric(metrics, "api_server_http_response_size_bytes")).ToNot(BeNil())
 	})
+
+	DescribeTable("inspect", func(inputFile string) {
```

Comment:
Does the fact that no test files have changed means the output for existing endpoints and resources is exactly the same?

Reply:
OK, yeah. you mentioned fixing the inspect API in this PR earlier today so I wasn't sure but I guess that was referring to the new endpoint